pkg/mimirpb/compat_test.go

```diff
@@ -452,6 +487,16 @@ func TestFromHistogramToHistogramProto(t *testing.T) {
 	assert.Equal(t, expected, p2)
 }
 
+func BenchmarkFromHistogramToHistogramProto(b *testing.B) {
```

Comment:
Is there a way to move the setup work out of the benchmark loop? Otherwise the benchmark will include the time taken in `GenerateTestHistogram` etc.